GameData/RealismOverhaul/Engine_Configs/RD0120T_Config.cfg

```diff
@@ -0,0 +1,173 @@
+//	Propellant: LOX / LH2
+//	Prop Ratio: 8.9% LH2, 11.1% Kerosene, 80.0% LOX
+//	Throttle: 100% to 38%
+//	Nozzle Ratio: 85.7
+//	Ignitions: 1
+//	=================================================================================
+//	RD-0120TP Mode 2
+//	Bipropellant mode
+//
+//	Dry Mass: 3800 Kg
+//	Thrust (SL): ??? kN
+//	Thrust (Vac): 735.3 kN
+//	ISP: ??? SL / 454 Vac
+//	Burn Time: 600
+//	Chamber Pressure: 8.14 MPa
+//	Propellant: LOX / LH2
+//	Prop Ratio: 5.95
+//	Throttle: 100% to 90%	guess, based on overall pressure ratio
+//	Nozzle Ratio: 85.7
+//	Ignitions: 1
+//	=================================================================================
+
+//	Sources:
+
+//	http://www.b14643.de/Spacerockets/Specials/Russian_Rocket_engines/engines.htm
+//	http://www.lpre.de/kbkha/RD-0120/index.htm
+//	http://www.astronautix.com/r/rd-0120.html
+//	https://ur.booksc.eu/book/66908213/dc76fb
+
+//	Used by:
+
+//	Notes:
+
+//	==================================================
+@PART[*]:HAS[#engineType[RD0120T]]:FOR[RealismOverhaulEngines]
+{
+	%title = RD-0120T
+	%manufacturer = KB Khimavtomatika
+	%description = 1990s medium TWR, atmospheric and vacuum use. The RD-0120T is a fuel-rich staged combustion tripropellant engine developed from the RD-0120, for use on SSTOs and other high performance vehicles. Tested in collaboration with Aerojet, but never flew.
+
+	@MODULE[ModuleEngines*]
+	{
+		%EngineType = LiquidFuel
+	}
+
+	!MODULE[ModuleEngineConfigs],*{}
+	!MODULE[ModuleAlternator],*{}
+	!RESOURCE,*{}
+
+	@MODULE[ModuleGimbal]
+	{
+		%gimbalRange = 11
+		%useGimbalResponseSpeed = true
+		%gimbalResponseSpeed = 16
+	}
+
+	MODULE
+	{
+		name = ModuleHybridEngine
+		type = ModuleEngines
+		origMass = 3.8
+		configuration = RD0120T-Mode-1
+		modded = false
+
+		CONFIG
+		{
+			name = RD0120T-Mode-1
+			minThrust = 687.7
+			maxThrust = 1809.8
+			heatProduction = 100
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.0647
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = LqdHydrogen
+				ratio = 0.6003
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = LqdOxygen
+				ratio = 0.3350
+				DrawGauge = False
+			}
+			atmosphereCurve
+			{
+				key = 0 418.7
+				key = 1 322.5
+			}
+			
+			ullage = True
+			ignitions = 1
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.5
+			}
+		}
+
+		CONFIG
+		{
+			name = RD0120T-Mode-2
+			minThrust = 661.8
+			maxThrust = 735.3
+			heatProduction = 100
+			PROPELLANT
+			{
+				name = LqdHydrogen
+				ratio = 0.729
+				DrawGauge = true
+			}
+			PROPELLANT
+			{
+				name = LqdOxygen
+				ratio = 0.271
+				DrawGauge = False
+			}
+			atmosphereCurve
+			{
+				key = 0 454
+				key = 1 350
+			}
+			
+			ullage = True
+			ignitions = 1
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.5
+			}
+		}
+	}
+}
+
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[RD-0120T-Mode-1]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
```

Comment:
Also need a TF config for RD0120T-Mode-2?

Reply:
TestFlight isn’t really aware of engine switching, it’ll just keep using the settings for the parent config, so multiple configs are unnecessary. This probably needs to be converted to MBEC though.